doc/programming_guide/ffd/deploying/configuration.rst

```diff
@@ -34,17 +34,17 @@ If options are changed, the application firmware must be rebuilt.
    * - appconfINTENT_UART_DEBUG_INFO_ENABLED
      - Enables/disables the UART intent debug information
      - 0
-   * - appconfINTENT_I2C_OUTPUT_ENABLED
-     - Enables/disables the |I2C| intent message
+   * - appconfI2C_MASTER_DAC_ENABLED
+     - Enables/disables configuring the DAC over |I2C| master
      - 1
-   * - appconfI2C_MASTER_ENABLED
-     - Enables/disables the |I2C| master mode to configure the DAC and send the intent message
+   * - appconfINTENT_I2C_MASTER_OUTPUT_ENABLED
+     - Enables/disables sending the intent message over |I2C| master
      - 1
-   * - appconfINTENT_I2C_OUTPUT_DEVICE_ADDR
+   * - appconfINTENT_I2C_MASTER_DEVICE_ADDR
      - Sets the address of the |I2C| device receiving the intent via the |I2C| master interface
      - 0x01
-   * - appconfI2C_SLAVE_ENABLED
-     - Enables/disables the |I2C| slave mode to read the device register with the intent message
+   * - appconfINTENT_I2C_SLAVE_POLLED_ENABLED
+     - Enables/disables allowing polling the intent message via |I2C| slave
```

Comment:
I suggest changing to `Enables/disables allowing another device to poll the intent message via |I2C| slave`

Reply:
Fixed in https://github.com/lucianomartin/sln_voice/pull/1/commits/e30e4603aca7a98dcbc5ec7757ef893a8840ee83.